hack/release/prepare-release.go

```diff
@@ -138,13 +138,17 @@ values:
 }
 
 func main() {
-	if len(os.Args) != 2 {
-		fmt.Printf("Usage: %s NEWVERS\n",
+	if len(os.Args) < 2 {
+		fmt.Printf("Usage: %s NEWVERS [OLDVERS]\n",
 			path.Base(os.Args[0]))
 		os.Exit(1)
 	}
 
 	newVers := os.Args[1]
+	oldVers := "master"
+	if len(os.Args) == 3 {
+		oldVers = os.Args[2]
+	}
```

Comment:
Let's switch the order of OLDVERS and NEWVERS, so that it is the same as for `make-release-tag.sh`.

Maybe something like this:
```Go

oldVers := master
newVers := ""

switch len(os.Args) {
case 2:
        newVers := os.Args[1]
case 3:
        oldVers := os.Args[1]
        newVers := os.Args[2]
default:
        fmt.Printf("Usage: %s NEWVERS | OLDVERS NEWVERS\n", ...)
}
```